demo/sprite.py

```
@@ -0,0 +1,57 @@
+
+"""Super class for all movable objects on the scene."""
+
+from abc import ABC
+from typing import Tuple
+
+import pygame
+
+from demo.direction import Direction
+
+
+class Sprite(ABC):
```

Ruff (0.12.2)

`Sprite` is an abstract base class, but it has no abstract methods or properties

(B024)